Dockerfile

```diff
@@ -11,10 +11,11 @@ ENV SGE_CONFIG_DIR=/usr/share/gridengine \
 
 RUN for PYTHON_VERSION in 2 3; do \
         export INSTALL_CONDA_PATH="/opt/conda${PYTHON_VERSION}" && \
-        . ${INSTALL_CONDA_PATH}/bin/activate && \
-        conda install -qy -n root drmaa && \
+        . /opt/conda/etc/profile.d/conda.sh && \
```

Comment:
Should have been using `INSTALL_CONDA_PATH` here. Fixing in PR ( https://github.com/jakirkham/docker_centos_drmaa_conda/pull/50 ).